client/test/index.js

```diff
@@ -77,7 +77,7 @@ require('nathanboktae-browser-test-utils');
 
 mocha.setup({
   ui: 'bdd',
-  globals: ['Scenario', 'testEl'],
+  globals: ['Scenario', 'testEl', '__VUE_DEVTOOLS_TOAST__'],
```

Comment:
is it a memory leak thing?

Reply:
Its not a memory leak, it just means there is part of the code which is assigning to a global variable (i.e. window.VARIABLE_NAME). Generally speaking this is bad practice in javascript as global variables can be changed by the end user which can create exploits and so on. This particular variable I don't think is an issue as it is used to turn on / off vue devtools so it won't ultimately cause issues within an application. There is another solution to just turn off dev tools when testing but I figure others in the open source community may come across this issue also.

See for more info (in Korean):
https://perlpark.github.io/articles/mocha-error-vue-devtools/